source/_docs/eu-region.md

```diff
@@ -9,59 +9,119 @@ earlyaccess: true
 earlynote: The documentation on this page discusses features and options that are not yet available to all users.
 ---
 
-Pantheon is extending early access invitations to contract customers who want to deploy a new site within the European Union (**EU**).
+Pantheon is extending early access to contract organizations (Enterprise, Reseller, OEM, and EDU+) that want to take a new site live from the European Union (**EU**) region.
 
-### Details
+## Enable Early Access
```

Comment:
"To learn more about enabling this feature, contact your dedicated account manager or [contact Us](https://pantheon.io/contact-us){.external}."